<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>421</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>rarity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>category</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>durability</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>weight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>attack</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>defense</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pB_add">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_del">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_change">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>430</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>change</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sB_del">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sB_change">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Сортировка</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="rB_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_rarity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>rarity</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_category">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>category</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_type">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>type</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_durability">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>durability</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_weight">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>weight</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_attack">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>attack</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_defense">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>defense</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_nothing">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>nothing</string>
     </property>
    </widget>
    <zorder>rB_type</zorder>
    <zorder>rB_name</zorder>
    <zorder>rB_rarity</zorder>
    <zorder>rB_category</zorder>
    <zorder>rB_durability</zorder>
    <zorder>rB_weight</zorder>
    <zorder>rB_attack</zorder>
    <zorder>rB_defense</zorder>
    <zorder>rB_nothing</zorder>
   </widget>
   <widget class="QPushButton" name="pB_statistic">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>540</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Статистика</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cB_characteristics_type">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>durability</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>weight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>attack</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>defense</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cB_statistic_type">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>540</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Минимальное</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Максимальное</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Суммарное</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Среднее</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pB_open_searchdialog">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>open search dialog</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_clearHighlights">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>clear highlights</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Работа с файлом</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="rB_saveToFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>сохранить</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_loadFromFile">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>взять</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_changePath">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>выбрать путь</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
